automatic/picard/picard.nuspec

```diff
@@ -19,14 +27,25 @@ Picard supports the majority of audio file formats, is capable of using audio fi
 When tagging files, Picard uses an album-oriented approach. This approach allows it to utilize the MusicBrainz data as effectively as possible and correctly tag your music. For more information, see the illustrated quick start guide to tagging.
 
 Picard is named after Captain Jean-Luc Picard from the TV series Star Trek: The Next Generation.
+
+## Features
+- Multiple Formats
+  Picard supports all popular music formats, including MP3, FLAC, OGG, M4A, WMA, WAV, and more.
```

Comment:
OK, didn't know that chocolatey uses hard breaks. The suggested format is the one which for example is used in the SourceTree package.